packages/components/src/tabs/styles.ts

```diff
@@ -13,12 +13,31 @@ import { space } from '../utils/space';
 import { reduceMotion } from '../utils/reduce-motion';
 
 export const TabListWrapper = styled.div`
+	position: relative;
```

Comment:
`isolation: isolate` is probably better here. I already found a bad interaction with a downstream override in the Preferences modal 🙁

https://github.com/WordPress/gutenberg/blob/f003f28c1d1b01d45a0b3455bd90cb671b7a993f/packages/preferences/src/components/preferences-modal-tabs/style.scss#L3-L4

Reply:
Hmm this should be fine though, the only override that'd break it is `position: static`, am I missing something?

Reply:
TIL, but `isolation: isolate` does not work. Seems like the stacking context is unrelated to positioning. According to MDN positioning is relative to the root OR the closest non-statically-positioned ancestor. So we need to stick with `position: relative`.